server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -663,3 +663,12 @@ def is_not_formatted(qcode):
                     {'qcode': formal_name, 'name': subject_codes.get(formal_name, ''), 'scheme': scheme})
 
         return formatted_subjects
+
+    def _plain_to_html(self, text):
+        # handle newline
+        for newline in ('\r\n', '\n', '\r'):
+            text = text.replace(newline, '</p><p>')
+        # remove redundant whitespaces
+        text = ' '.join(text.split())
+
+        return '<p>' + text + '</p>'
```

Comment:
if you convert plain text to html, I guess you need to escape the text, what if a `<` or `>` is present else?
Also it would probably make sense to move this method to some generic place so it can be re-used, for instance in `superdesk/etree.py`